### DISCUSSION TITLE
Unable to click open URL's

### DISCUSSION THREAD

**Discussion by ghost** [kovidgoyal/kitty#5880] [Q&A]
I have kitty installed on parrot using mate desktop running in parallels. I can’t seem to get URLs to open by clicking on them. Open with url default is enabled. My xdg-open is set to Firefox. I can open them with shift+ctrl+e but not by clicking on them. Can anyone help me figure this out please?

**Comment by SolitudeSF** [comment]
in what context are urls displayed? if you see them in text editor with mouse support it will consume mouse clicks.

**Reply by ghost** [reply]
So before I had installed  kitty through apt install. It installed an older version. I updated to the second most recent one here on github, the most recent one requires glibc 2.3.4 which parrot is not running on. That solved the problem.
